operator/v1/types_authentication.go

```diff
@@ -7,6 +7,8 @@ import (
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +openshift:compatibility-gen:level=4
+// +openshift:compatibility-gen:internal
```

Comment:
why?

Reply:
For all of them in here.

Reply:
These types are internal to the operators and can change at any moment. None of them should be exposed (I am trying to see if I can [add a test](https://github.com/openshift/origin/pull/25947) for this.)

Reply:
They are not internal. These must be

```
// +openshift:compatibility-gen:level=1
```

Reply:
Deferring this to a followup PR. Skipping for now. Also, see: https://github.com/openshift/api/pull/803#discussion_r553551318